.github/workflows/e2e-upgrade.yaml

```diff
@@ -12,7 +9,16 @@ jobs:
       matrix:
         include:
           - test: TestV4ToV5ChainUpgrade
+            chain-image: ghcr.io/cosmos/ibc-go-simd
+            chain-a-tag: v4.0.0
+            chain-b-tag: v4.0.0
+            chain-upgrade-tag: pr-2144 # TODO: backport default upgrade handler to release/v5.x.x
           - test: TestV5ToV6ChainUpgrade
+            chain-image: ghcr.io/cosmos/ibc-go-icad
+            chain-binary: icad
+            chain-a-tag: v0.3.5
+            chain-b-tag: v0.3.5
+            chain-upgrade-tag: v0.4.0
```

Comment:
should this be v6.0.0?

Reply:
this is referring to the `icad` docker image version, which includes `ibc-go/v6-alpha1`